src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useRef } from 'react';
+import styles from './Filters.module.css';
+import Button from '../Button';
+import { CheckboxField, FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
+import Icon from '../Icon';
+
+export type Props = {
+  /**
+   * Checkbox fields that will be displayed in the filters drawer.
+   */
+  checkboxFields: CheckboxField[];
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Callback called when filters drawer is closed.
+   */
+  closeFilters: (checkedValues: { [key: string]: boolean }) => void;
```

Comment:
Could this be called something like `onClose`?

Reply:
Yup, that would be more intuitive and follow other prop patterns